user/tests/wiring/no_fixture/cellular.cpp

```diff
@@ -157,6 +157,56 @@ test(CELLULAR_07_rssi_is_valid) {
     assertMoreOrEqual(s.rssi, -150);
 }
 
+test(CELLULAR_08_sigstr_is_valid) {     // collects signal strength and quality
+    connect_to_cloud(6*60*1000);
+    assertTrue(Particle.connected());
+    CellularSignal s;
+    for (int x = 0; x < 10; x++) {
+        s = Cellular.RSSI();
+        if (s.getStrengthValue() <= 0 && s.getQualityValue() <= 0) {
+            break;
```

Comment:
I would actually perform the check here multiple times.

Reply:
Changed!